webserver/jersey/src/main/java/io/helidon/webserver/jersey/ResponseWriter.java

```diff
@@ -105,19 +65,14 @@ public void flush() throws IOException {
     @Override
     public OutputStream writeResponseStatusAndHeaders(long contentLength, ContainerResponse context)
             throws ContainerException {
-
-        //
-        // TODO also check that nothing was written an nothing was read
-        //
         if (context.getStatus() == 404 && contentLength == 0) {
             whenHandleFinishes.thenRun(() -> {
                 LOGGER.finer("Skipping the handling and forwarding to downstream WebServer filters.");
-
                 req.next();
             });
             return new OutputStream() {
                 @Override
-                public void write(int b) throws IOException {
+                public void write(int b) {
```

Comment:
(This could be a static class.)

Reply:
This could be `OutputStream.nullOutputStream();`